esmvalcore/config-user.yml

```diff
@@ -45,19 +72,30 @@ profile_diagnostic: false
 # For site-specific entries, see below
 # Comment out these when using a site-specific path
 rootpath:
-  default: ~/climate_data
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4MIPs: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6: ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
 
 # Directory structure for input data: [default]/ESGF/BADC/DKRZ/ETHZ/etc
 # See config-developer.yml for definitions.
 # comment out/replace as per needed
 drs:
-  CMIP3: ESGF
-  CMIP5: ESGF
-  CMIP6: ESGF
-  CORDEX: ESGF
-  obs4MIPs: ESGF
+  CMIP3: default
+  CMIP5: default
+  CMIP6: default
+  CORDEX: default
+  obs4MIPs: default
```

Comment:
What was the reason for changing this back to the old default from before #1217? I liked the new default value, because it encourages storing the data in such a structured way, that enables you to find out what version of the files you have. If you download CMIP data and store it with the `default` (i.e. no subdirectories) format, it becomes very hard to tell what version of the files you have.

Reply:
yes but that assumes an ESGF-like structure (ie you have downloaded data) - we should NOT encourage people to download data

Reply:
See comment above